_rules/meta-viewport-b4f0c3.md

```diff
@@ -0,0 +1,225 @@
+---
+id: b4f0c3
+name: meta viewport does not prevent zoom
+rule_type: atomic
+description: |
+  This rule checks that the `meta` element is not used to block the user agent ability to zoom. 
+accessibility_requirements:
+  wcag21:1.4.4: # Resize text (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+authors:
+  - Jey Nandakumar
+  - Audrey Maniez
+---
+
+## Applicability
+
+The rule applies to each [`<meta>`](https://www.w3.org/TR/html52/document-metadata.html#the-meta-element) element with a [`name`](https://www.w3.org/TR/html52/document-metadata.html#element-attrdef-meta-name) attribute whose value is set to [`viewport`](https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag#Enter_viewport_meta_tag) and has a [`content`](https://www.w3.org/TR/html52/document-metadata.html#dom-htmlmetaelement-content) attribute.
+
+## Expectation
+
+The [`content`](https://www.w3.org/TR/html52/document-metadata.html#dom-htmlmetaelement-content) attribute, whose value is mapped to a list of property/value pairs in a user-agent specific manner, does not:
+
+- specify the property `user-scalable` with a value of `yes`; nor
+- specify the property `maximum-scale` with a value of less than 2
+
+**Note:** 
+1) For Safari on the iPhone, the content attribute is mapped to property/value pairs according to the following algorithm: https://drafts.csswg.org/css-device-adapt/#parsing-algorithm
+2) Properties and the values for the content attribute of `<meta name="viewport">` are specified in a detailed tabulation [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta#Attributes).
```

Comment:
The anchor "here" will either need a proper name or the link will have to be moved:

> [...] here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta#Attributes

Reply:
Unfortunately there is no direct link to the tabulation I wanted to refer & background already refers to `meta` element page. So removing note 2.